account_debitnote/README.rst

```diff
@@ -0,0 +1,88 @@
+===========
```

Comment:
You can remove this file. I will be create automatically once it is merged.

Reply:
Okay, I'll remove it.

> We should not allow create Debit Note from Debit Note. Please hide "Create Debit Note" button, from the debit note itself. Also check the same in code.
> Debit Note should be create from Invoice only.
> 
> Steps:
> 
> * From an Incoie, INV-001, I can create Debit Note, i.e., DINV-001 -> OK
> * From Debit Note, i.e., DINV-001, I can create Debit Note, i.e., DINV-002 -> Not OK.

Okay, I'll fix it